src/hooks/useUser.ts

```diff
@@ -6,6 +6,7 @@ import {
 } from '../../server/lib/permissions';
 import { UserType } from '../../server/constants/user';
 import { mutateCallback } from 'swr/dist/types';
+import { UserSettings } from '../../server/entity/UserSettings';
```

Comment:
Let's not cross import this. We don't cross import `User` because it can cause compilation issues. So let's keep rebuilding the user type here manually.

Reply:
Got it!